packages/currency/src/index.js

```diff
@@ -69,3 +69,13 @@ export function getCurrencyFormatString( number ) {
 	}
 	return number.toFixed( precision );
 }
+
+export function renderCurrency( number, currencySymbol ) {
```

Comment:
Seems like a good follow up would be to make this a component that recognizes negative formats, as @timmy mentuioned,

Reply:
>So you think this should be its own component?

I do. A good use case is extensions creating a report page with tables displaying currency values. I can see this being really useful as we build out other screens in the future.

Reply:
Okay, sounds good @psealock!  Opened an issue to handle this component [here](https://github.com/woocommerce/woocommerce-admin/issues/2202).  We can also handle wrapping other currency values in that PR to keep scope down on this one.